packages/twenty-server/src/modules/workflow/common/standard-objects/workflow-run.workspace-entity.ts

```diff
@@ -32,17 +32,21 @@ export enum WorkflowRunStatus {
   FAILED = 'FAILED',
 }
 
-export type WorkflowRunOutput = {
-  steps: {
-    id: string;
-    name: string;
-    type: string;
+type StepRunOutput = {
+  id: string;
+  name: string;
+  type: string;
+  outputs: {
     attemptCount: number;
     result: object | undefined;
     error: string | undefined;
   }[];
 };
 
+export type WorkflowRunOutput = {
+  steps: Record<string, StepRunOutput>;
```

Comment:
The consequence of storing the output of the steps in an object is that their order is not guaranteed to be preserved. That's not an issue on its own, but the frontend will have to fetch the steps definition to print the output in order.